judgels-backends/uriel/uriel-app/src/test/java/judgels/uriel/contest/clarification/ContestClarificationStoreTests.java

```diff
@@ -0,0 +1,72 @@
+package judgels.uriel.contest.clarification;
+
+import java.util.Optional;
+import judgels.persistence.api.Page;
+import judgels.persistence.api.SelectionOptions;
+import judgels.uriel.persistence.ContestClarificationDao;
+import judgels.uriel.persistence.ContestClarificationModel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+class ContestClarificationStoreTests {
```

Comment:
So, it is intentional that in Judgels, we don't have unit tests for store layer. Because this does not actually test anything -- we don't even test that the Hibernate DAO is returning correct DB rows (because we mock it).

Can we add the tests to `ContestClarificationStoreIntegrationTests` instead? Here, we actually run `ContestClarificationDao` against real (in-memory) DB.

Reply:
Hmmm, okay, I can add to the integration test.
I was thinking the unit test can help prevent regression since it's changing parameter and logic of existing method.

Reply:
Updated, removed the unit test.